README.md

```diff
@@ -3,3 +3,19 @@
 See documentation at https://merge.dev/docs/linking-flow/get-started.
 
 When updating this repo, bump the version number in `package.json` according to this semantic versioning scheme: https://docs.npmjs.com/about-semantic-versioning
+
+---
+
+### Frequently Asked Questions (FAQ)
+
+#### **Q: What version of Vue do you support?**
+**A:** The latest version of this package requires Vue version 3.0+. If you need support for an older version of Vue, you can use an older version of this package (1.1). Note that these packages are not supported and older versions of Vue have been [deprecated](https://vuejs.org/about/faq.html#is-vue-2-still-supported).
```

Comment:
nit - mb its just me but saying "these packages are not supported" kinda reads as this vue-merge-link package may not be supported anymore mb specify these versions are not supported

not a strong feeling but just what i got when i read through it

Reply:
Yeah good call i'll reword it